jbmc/src/java_bytecode/java_static_initializers.cpp

```diff
@@ -52,8 +52,8 @@ static typet clinit_states_type()
 
 // Disable linter here to allow a std::string constant, since that holds
 // a length, whereas a cstr would require strlen every time.
-const std::string clinit_wrapper_suffix = "::clinit_wrapper"; // NOLINT(*)
-const std::string user_specified_clinit_suffix = "::user_specified_clinit"; // NOLINT(*)
+const std::string clinit_wrapper_suffix = ".<clinit_wrapper>"; // NOLINT(*)
```

Comment:
:question: Why are you fixing in this PR?

Reply:
Because I would have to special-case it in my tests. I could put this change into a separate PR that goes before this one here.

Reply:
No need on my behalf :slightly_smiling_face: